tests/end2end/features/spell.feature

```diff
@@ -0,0 +1,30 @@
+# vim: ft=cucumber fileencoding=utf-8 sts=4 sw=4 et:
+
+Feature: Setting spell checking for QtWebEngine
+
+    @qtwebkit_skip @qt>=5.8
+    Scenario: Turn spell check on
+        Given I set spellcheck.enabled to false
+        When I run :set spellcheck.enabled true
+        Then the option spellcheck.enabled should be set to true
+
+    @qtwebkit_skip @qt>=5.8
+    Scenario: Turn spell check off
+        Given I set spellcheck.enabled to true
+        When I run :set spellcheck.enabled false
+        Then the option spellcheck.enabled should be set to false
+
+    @qtwebkit_skip @qt>=5.8
+    Scenario: Set an invalid language
+        When I run :set spellcheck.languages ['invalid-language'] (invalid command)
+        Then the error "set: Invalid value 'invalid-language' *" should be shown
```

Comment:
The tests above only test the normal setting machinery and nothing specific to your changes really - so I'd drop them.